### PR TITLE
feat: added rating component

### PR DESCRIPTION
**Pull request by TejasShekar**
1. Added star rating component

Screenshot 2022-02-19 151257

### CODE REVIEW

File: Components/Rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.rating-container input {
+  display: none;
+}
+
+.rating-container label {
+  cursor: pointer;
+  color: gold;
```
Comment (zabihshaik) [inline Components/Rating/rating.css]: Better to have variable names if the colors are repeated.

**Review by zabihshaik — APPROVED** [review]
Everything looks good.

File: Components/Rating/rating.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+}
+
+.rating-container input:not(:checked) + label {
+  color: grey;
```
Comment (praveenreddy1798) [inline Components/Rating/rating.css]: use colors from color variable instead of declaring them directly

File: Components/Rating/rating.css (hunk 1 above)
Comment (TejasShekar) [reply Components/Rating/rating.css]: Done

File: Components/Rating/rating.css (hunk 2 above)
Comment (TejasShekar) [reply Components/Rating/rating.css]: Done

**Review by praveenreddy1798 — APPROVED** [review]

**Pull request merged**